cpp/roaring.hh

```diff
@@ -499,6 +499,17 @@ public:
         return api::roaring_bitmap_rank(&roaring, x);
     }
 
+    /**
+     * Returns the index of x in the set, index start from 0.
+     * If the set dosen't contain x , this function will return -1.
+     * The difference with rank function is that this function will return -1
+     * when x isn't in the set, but the rank funciton will return a
+     * non-negative number.
+     */
+    int64_t get_index(uint32_t x) const noexcept {
```

Comment:
The other C++ methods are `camelCase`, I think this should be `getIndex`

Do we want to add the same to `roaring64map.hh`?